src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -197,6 +197,16 @@ class DefaultStorageKeyAllocator
         return StorageKeyName::Formatted("g/su/%x", static_cast<unsigned>(index));
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
+    // Scene Storage
+    static StorageKeyName SceneFabricList()
+    {
+        return StorageKeyName::FromConst("g/gfl");
+    } // shares key with group fabric list to minimize flash usage
+    static StorageKeyName FabricScenesKey(chip::FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/s", fabric); }
```

Comment:
Is this a FabrcScenesIndexKey ? As in, is this for an index?

Reply:
It's used for FabricSceneData, which gets a map the the scenes in fabric, a scene count and the index of the next fabric. Will be renamed to "FabricSceneDataKey"